### ISSUE TITLE
bug with hundreds

### ISSUE THREAD

**Issue by gevorg95** [KurroLopez/CosNumberTranslate#6]
Hello!

I encountered the following mistake:
```
> w ##class(NumberTranslate.NumberTranslate).GetText(1250,.tSc,"en")
> thousand two hundred and fifty
> w ##class(NumberTranslate.NumberTranslate).GetText(1200,.tSc,"en")
> thousand hundred
```

**Comment by KurroLopez** [comment]
Thanks for your issue. I'll have a look ASAP

**Comment by KurroLopez** [comment]
Fixed issue into versión 1.0 and 1.1
